### DISCUSSION TITLE
Arduino UNO Wifi Rev 2 - OTA possible?

### DISCUSSION THREAD

**Discussion by AdamWithHap** [JAndrassy/ArduinoOTA#188] [General]
Hi Juraj,

I hope you are well.

I was wondering if you knew of a way to flash sketches OTA to an Arduino Uno WIFI REV 2.  There seem to bit hints that it might be possible with WifiNINA libraries but I can't seem to find any more information.

Thank you,

Adam

**Comment by JAndrassy** [comment]
no. ATmega 4809 is not supported

**Reply by AdamWithHap** [reply]
Hi Juraj,

Thanks for the reply.

Adam
